src/briefcase/integrations/subprocess.py

```diff
@@ -413,14 +416,18 @@ def _stream_output_thread(self, popen_process):
 
         :param popen_process: popen process to stream stdout
         """
-        while True:
-            # readline should always return at least a newline (ie \n)
-            # UNLESS the underlying process is exiting/gone; then "" is returned
-            output_line = ensure_str(popen_process.stdout.readline())
-            if output_line:
-                self.command.logger.info(output_line)
-            elif output_line == "":
-                return
+        # ValueError is raised if stdout is unexpectedly closed.
+        # This can happen if the user starts spamming CTRL+C, for instance.
+        # Silently exit to avoid Python printing the exception to the console.
+        with contextlib.suppress(ValueError):
```

Comment:
I found this while experimenting with sending CTRL+C while building an AppImage in Docker. Different versions of Python manifest the error differently in the console but they all raise `ValueError`.